### PR TITLE
Bug 1782153: templates: Add a "this is generated" note

### PR DESCRIPTION
**Pull request by vrutkovs**
This is a best practice, and particularly in this case
when we're intending to modify a config file, it can't be identical
to the shipped version.

This avoids having libostree replace it with a newer version.

Cherry-pick of #1320 on release-4.1.

### CODE REVIEW

**Comment by cgwalters** [comment]
You'll need to run a command to update the test copies of the templates, `git log` should find it.

File: templates/worker/01-worker-container-runtime/_base/files/container-storage.yaml [hunk 1]
```diff
@@ -117,4 +120,3 @@ contents:
     # attempt to complete IO when ENOSPC (no space) error is returned by
     # underlying storage device.
     # xfs_nospace_max_retries = "0"
-
```
Comment (kikisdeliveryservice) [inline templates/worker/01-worker-container-runtime/_base/files/container-storage.yaml]: can we put this newline back?

File: templates/master/01-master-container-runtime/_base/files/container-storage.yaml [hunk 2]
```diff
@@ -117,4 +120,3 @@ contents:
     # attempt to complete IO when ENOSPC (no space) error is returned by
     # underlying storage device.
     # xfs_nospace_max_retries = "0"
-
```
Comment (kikisdeliveryservice) [inline templates/master/01-master-container-runtime/_base/files/container-storage.yaml]: can we put this newline back?

Comment (vrutkovs) [reply templates/master/01-master-container-runtime/_base/files/container-storage.yaml]: Done, thanks!

**Comment by kikisdeliveryservice** [comment]
thanks @vrutkovs !

/appprove

will let @cgwalters lgtm...

**Comment by kikisdeliveryservice** [comment]
err... 

/approve

**Comment by cgwalters** [comment]
/approve
Thanks for doing this!

**Pull request merged**